### DISCUSSION TITLE
weeklies/8

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#136] [Comments]
https://hutusi.com/articles/weekly-8 

 本文为周刊第八期，前期的周刊参见：周刊系列。

**Comment by ColarXu** [comment]
这一期每篇文章，我都很感兴趣，明天去把原文也看了。感谢分享。

**Comment by ColarXu** [comment]
对，忘了说，有个错别字。

第二篇文章，正文第二段的第一句：HN 于 2007 年面试。

里面的「面试」应该为「面世」。

**Comment by hutusi** [comment]
@ColarXu 谢谢~~ typo fixed.

很开心写的东西对大家有帮助 😊
